src/main/webapp/stories.html

```diff
@@ -48,6 +48,16 @@ <h4 class="pb-2 mb-1">Ninetailed Personalization Platform</h4>
         </div>
     </div>
 
+    <div class="row m-5">
+        <div class="col">
+            <h4 class="pb-2 mb-1"> Dropsolid - The Digital Experience Company</h4>
+            <p>The Dropsolid experience cloud is not only a cloud platform that allows developers and marketers to work faster and without restrictions of proprietary code, it also delivers an out of box DXP ready enhanced Drupal CMS, Mautic marketing automation platform and Unomi CDP to build powerful user experiences.</p>
```

Comment:
Would it be possible to tone down the marketing a little and instead talk a little more about how Apache Unomi is integrated into the different pieces? I try to ask people to stay as factual as possible to avoid breaking Apache Foundation "no ads" rule.

Reply:
Is it better like this?